pkg/scheduler/apis/config/types.go

```diff
@@ -105,6 +105,16 @@ type KubeSchedulerProfile struct {
 	// is scheduled with this profile.
 	SchedulerName string
 
+	// PercentageOfNodesToScore is the percentage of all nodes that once found feasible
+	// for running a pod, the scheduler stops its search for more feasible nodes in
+	// the cluster. This helps improve scheduler's performance. Scheduler always tries to find
+	// at least "minFeasibleNodesToFind" feasible nodes no matter what the value of this flag is.
+	// Example: if the cluster size is 500 nodes and the value of this flag is 30,
+	// then scheduler stops finding further feasible nodes once it finds 150 feasible ones.
+	// When the value is 0, default percentage (5%--50% based on the size of the cluster) of the
```

Comment:
We should add here too that when null the value is taken from the global field.

Reply:
How about `If it is not set (nil), the global PercentageOfNodesToScore will be used.`?

Reply:
Just copy the comment from the v1 package :)

Reply:
Updated the comment.